### DISCUSSION TITLE
What is the business model behind EdgeDB?

### DISCUSSION THREAD

**Discussion by joepie91** [geldata/gel#3377] [General]
I've been looking into EdgeDB a bit, as it seems quite interesting conceptually, but there's something I've not been able to find an answer to: what's the business model of the company behind it? Clearly, EdgeDB Inc. is a for-profit corporation that is hiring developers and other folks to work on it, and that money needs to come from *somewhere* eventually, once the VC funding runs out.

I'm asking because over the past several years, quite a few VC-funded for-profit "open-source" projects (including databases) have switched to proprietary licenses such as the BSL, presumably because they couldn't produce enough ROI on an added-services business model. While understandable from a business perspective, that sort of license change is a significant problem for open-source projects, which are suddenly hard-depending on a proprietary component that they can't easily move away from/

What's the plan, if any, to prevent this from happening to EdgeDB? What's the envisioned business model to produce the ROI that the investors undoubtedly expect eventually, and do the founders and other people running the company retain enough control in the company to make assurances about that?

(Decided to post this in General rather than Q&A, seeing as this is not exactly a technical question asked of the community)

**Comment by 1st1** [comment]
> I’ve been looking into EdgeDB a bit, as it seems quite interesting conceptually, but there’s something I’ve not been able to find an answer to: what’s the business model of the company behind it? Clearly, EdgeDB Inc. is a for-profit corporation that is hiring developers and other folks to work on it, and that money needs to come from somewhere eventually, once the VC funding runs out.

This is a fair question and it has a straightforward answer: we will run and support EdgeDB infrastructure for you (a managed EdgeDB service is coming).

> I’m asking because over the past several years, quite a few VC-funded for-profit “open-source” projects (including databases) have switched to proprietary licenses such as the BSL, presumably because they couldn’t produce enough ROI on an added-services business model.
> <..>
> What’s the plan, if any, to prevent this from happening to EdgeDB?

I cannot disclose any specific company plans here yet, but I can say that we are acutely aware of this trend and are actively working on building a deep enough moat to keep the core of EdgeDB open and free.  Some parts of this moat would necessarily be proprietary non-trivial value-adds, but also a large part of the strategy lies with the fact that we are not just building a commodity database server sitting behind an IP address, we are building a *developer experience*, which would be a lot harder to replicate for a would-be cloud hosting competitor that is used to methodically amalgamating software into a grey mass of “cloud services”.

**Reply by jedihugo** [reply]
Yass! Managed service please! 🚀
